### ISSUE TITLE
Empty PSI element Error in Plugin

### ISSUE THREAD

**Issue by orklah** [funivan/PhpClean#16]
Hi, I just got an error in the Plugin reported by PHPStorm.

It is reproductible with the following code:
```php
<?php

class test
{

    public function a(<caret>){

    }
}
```
Type the character "," in the caret position

Here is the complete error log:

> java.lang.Throwable: Empty PSI elements must not be passed to createDescriptor. Start: Parameter, end: Parameter, startContainingFile: PHP file
> 	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:134)
> 	at com.intellij.codeInspection.ProblemDescriptorBase.<init>(ProblemDescriptorBase.java:61)
> 	at com.intellij.codeInspection.InspectionManagerBase.createProblemDescriptor(InspectionManagerBase.java:83)
> 	at com.intellij.codeInspection.InspectionManagerBase.createProblemDescriptor(InspectionManagerBase.java:72)
> 	at com.intellij.codeInspection.ProblemsHolder.registerProblem(ProblemsHolder.java:66)
> 	at com.intellij.codeInspection.ProblemsHolder.registerProblem(ProblemsHolder.java:59)
> 	at com.funivan.idea.phpClean.inspections.missingParameterType.MissingParameterTypeDeclarationInspection$buildVisitor$1.visitPhpMethod(MissingParameterTypeDeclarationInspection.kt:27)
> 	at com.jetbrains.php.lang.psi.elements.impl.MethodImpl.accept(MethodImpl.java:89)
> 	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:75)
> 	at com.intellij.codeInspection.InspectionEngine.createVisitorAndAcceptElements(InspectionEngine.java:63)
> 	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.runToolOnElements(LocalInspectionsPass.java:292)
> 	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$null$5(LocalInspectionsPass.java:260)
> 	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:126)
> 	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:115)
> 	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$6(LocalInspectionsPass.java:257)
> 	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:133)
> 	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1163)
> 	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:105)
> 	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:582)
> 	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
> 	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:87)
> 	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:116)
> 	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:96)
> 	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:147)
> 	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:222)
> 	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:96)
> 	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
> 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
> 	at java.util.concurrent.ForkJoinPool$WorkQueue.pollAndExecCC(ForkJoinPool.java:1190)
> 	at java.util.concurrent.ForkJoinPool.helpComplete(ForkJoinPool.java:1879)
> 	at java.util.concurrent.ForkJoinPool.awaitJoin(ForkJoinPool.java:2045)
> 	at java.util.concurrent.ForkJoinTask.doJoin(ForkJoinTask.java:390)
> 	at java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:719)
> 	at java.util.concurrent.ForkJoinPool.invoke(ForkJoinPool.java:2616)
> 	at com.intellij.concurrency.JobLauncherImpl.invokeConcurrentlyUnderProgress(JobLauncherImpl.java:65)
> 	at com.intellij.concurrency.JobLauncher.invokeConcurrentlyUnderProgress(JobLauncher.java:56)
> 	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.visitPriorityElementsAndInit(LocalInspectionsPass.java:263)
> 	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.inspect(LocalInspectionsPass.java:196)
> 	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.collectInformationWithProgress(LocalInspectionsPass.java:115)
> 	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
> 	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:69)
> 	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:423)
> 	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1168)
> 	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:416)
> 	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:582)
> 	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
> 	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:87)
> 	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:415)
> 	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:391)
> 	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:147)
> 	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:222)
> 	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:389)
> 	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:161)
> 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
> 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
> 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
> 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)

**Comment by funivan** [comment]
Thanks for the report. I will check it.

**Comment by funivan** [comment]
Fixed version 2019.01.28
